whole-tale/ngx-dashboard
# fix: allow to pass config via env vars. Fixes #145

## Problem

Some urls / variables needs to be configurable via env vars.

## Approach
https://pumpingco.de/blog/environment-variables-angular-docker/

## How to Test

1. Deploy with deploy-dev (note: GIRDER_URL_API should point to `https://girder.local.wholetale.org/api/v1` whereas currently in deploy-dev@master it points to `https://girder.local.wholetale.org`. Unless you change it, you'll get a dashboard's blue screen of death. That's also how you can test this :) )
2. (Optionally) Deploy on .stage.local.wholetale.org (build and push image, or use wholetale/ngx-dashboard:env)
2. Login to the WholeTale Dashboard
